hello-world-notification-microservice/README.md

```diff
@@ -1,26 +1,30 @@
-##Example Notification 2.0 Microservice
-This example microservice shows you how to create a subscription & consume a stream of notifications using WebSocket protocol. Before running this example microservice, please make sure to have created a device and specified its ID in `example.source.id` property located in applications.properties. As part of cleanup, you may wish to delete this device after you have finished running this example. The example microservice first creates a subscription and then proceeds to connect to the WebSocket server after obtaining the authorization token.
+# Introduction
+This example microservice demonstrates how to create a subscription and consume a stream of notifications using WebSocket protocol. The example microservice first creates a subscription and then proceeds to connect to the WebSocket server after obtaining the authorization token.
 
 The example microservice creates the following subscription to the specified device:
```

Comment:
Might be useful to indicate where the user might specify such a device.

Reply:
Not sure what you mean here - that it needs to be specified in application.properties? The guide below covers that, I didn't want to repeat information in the Introduction

Reply:
how about "create a subscription for notifications about a device"... "Any device could be used but we show how to create simulate a device for testing with this microservice below."